addons/mail/security/mail_security.xml

```diff
@@ -276,4 +276,32 @@
             <field name="perm_create" eval="True"/>
             <field name="perm_unlink" eval="True"/>
         </record>
+
+        <record id="ir_rule_mail_canned_response_admin_rule" model="ir.rule">
+            <field name="name">Canned response: admin access</field>
+            <field name="model_id" ref="model_mail_canned_response"/>
+            <field name="groups" eval="[Command.link(ref('group_mail_canned_response_admin'))]"/>
+            <field name="perm_read" eval="False"/>
```

Comment:
It doesn't make a lot of sense to restrict "read" to admin.

Reply:
It's because we don't want the admin to see all the canned responses (of all users). Maybe there is a better way to do it?

Reply:
This shouldn't be done with ACL, it should be a specific domain on the places that need restriction.